src/client/modules/content/widgets/data_table/DataTable.jsx

```diff
@@ -121,10 +126,12 @@ export default class DataTable extends React.Component {
         }
 
         return (
-
-                <div className="box">
+                <div className="box" style={this.props.style}>
                     <div className="box-header">
-                            <h3 className="box-title">{this.props.tableName}</h3>
+
+                        <h3 className="box-title">
+                            <i className={`fa fa-${this.props.tableIcon}`}/> {` ${this.props.tableName}`}
```

Comment:
Typically the property would be `iconClass` and contain the full class specification, `'fa fa-...'`.